source/extensions/common/proxy_protocol/proxy_protocol_header.cc

```diff
@@ -118,6 +127,27 @@ void generateV2LocalHeader(Buffer::Instance& out) {
   out.add(addr_fam_protocol_and_length, 4);
 }
 
+void generateV2Header(const Network::ProxyProtocolData& prox_proto_data, Buffer::Instance& out) {
+  uint16_t extension_length = 0;
+  for (auto& tlv : prox_proto_data.tlv_vector_) {
+    // TODO: handle overflow without crashing.
+    // assert(tlv.value.size() + 3u < std::numeric_limits<uint16_t>::max() - extension_length);
+    extension_length += 3 + tlv.value.size();
```

Comment:
Can you add a comment or constant to help explain the 3 numeric literal?

Reply:
Good point! Done.